### PR TITLE
docs: align formatting with vite docs

### PR DESCRIPTION
**Pull request by sheremet-va**

### CODE REVIEW

File: docs/api/index.md [hunk 1]
```diff
@@ -14,176 +14,175 @@ For compatibility with Jest, `TestFunction` can also be of type `(done: DoneCall
...
+  import { test, expect } from 'vitest'
 
-test('should work as expected', () => {
-  expect(Math.sqrt(4)).toBe(2);
-})
-```
+  test('should work as expected', () => {
+    expect(Math.sqrt(4)).toBe(2);
+  })
+  ```
```
Comment (patak-dev) [inline docs/api/index.md:28]: Some strange happened with formatting here

Comment (patak-dev) [reply docs/api/index.md:28]: Nvm, ignore

**Review by patak-dev — APPROVED** [review]

File: docs/api/index.md (hunk 1 above)
Comment (sheremet-va) [reply docs/api/index.md:28]: what do you mean? this is how vite code docs are formatted: https://raw.githubusercontent.com/vitejs/vite/main/docs/config/index.md

Comment (patak-dev) [reply docs/api/index.md:28]: I missread the diff, thus the nvm, ignore the noise 😁

Comment (dammy001) [reply docs/api/index.md:28]: Yeah. 😂

**Pull request merged**